pegasus/sites.v3/code.org/public/css/brstrap.css

```diff
@@ -57,7 +57,7 @@
   .col-25 { width: 25%; float:left; }
   .col-26 { width: 26%; float:left; }
   .col-32 { width: 32%; float:left; }
-  .col-33 { width: 33%; float:left; }
+  .col-33 { width: 33.33333333%; float:left; }
```

Comment:
out of curiosity, why is this precision needed here?

Reply:
The three panels (left image, center hero, right image) on the homepage were actually falling short of the right-hand edge of the screen, and I discovered it was because we were losing pixels by not counting them with the original `33%`.